tests/run.bats

```diff
@@ -98,7 +98,7 @@ load helpers
 	[ "$output" = that-other-thing ]
 
 	buildah config --entrypoint echo $cid
-	run buildah --debug=false run $cid that-other-thing
+	run buildah --debug=false run $cid echo that-other-thing
```

Comment:
This test is going away with #596.  Can you revert this change to avoid a merge conflict?

Reply:
Or if the test is causing your PR issues, just remove it in yours too.

Reply:
Shouldn't this test remain to ensure that `buildah run` doesn't use the `entrypoint` value, which is the desired behaviour?